config/storage.yml

```diff
@@ -5,3 +5,20 @@ test:
 local:
  service: Disk
  root: <%= Rails.root.join("storage") %>
+
+# Main service to be used for staging - also backs up to prod
+staging:
+  service: S3
+  access_key_id: <%= ENV["AWS_ACCESS_KEY_ID"] %>
+  secret_access_key: <%= ENV["AWS_SECRET_ACCESS_KEY"] %>
+  region: us-east-1
```

Comment:
@stanleypliu I think this and the region on production config need to change now too, right?